notebooks/image_models/solutions/deepconv_gan.ipynb

```diff
@@ -50,33 +50,6 @@
     "## Import TensorFlow and other libraries"
```

Comment:
<p><code>noise_dim</code> and <code>num_examples_to_generate</code> in upper caps  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Agreed. Thanks!</p>